### PR TITLE
Github deploy

### PR DESCRIPTION
**Pull request by hislordshipprof**
-Create pages and deploy using github pages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ npm init -y or (npm install --save-dev hint@6.x)
 npm start
 
 
-And you can start demo via `https://localhost:5500`
+And you can start demo via `https://hislordshipprof.github.io/Hello-Microverse/`
```
Comment (vechicin) [inline README.md]: I recommend you provide an actual link to the website. In order to do so you need to wrap the text you want to link in [] and then paste the actual link right after the brackets inside (). Please fix this so users that take a look at your project can directly go to the live demo without having to copy and paste the link.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @hislordshipprof  ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Proper GitHub Pages deployment

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by vechicin** [comment]
https://github.com/hislordshipprof/Hello-Microverse/blob/0091dd5bb94d31a0b9c12f3f57a5b7bc7b9b54aa/README.md?plain=1#L46-L49

Please link your social media links properly and change the handles from generic ones to your actual handles in order to give yourself more visibility and make your documentation look more professional.

**Comment by vechicin** [comment]
https://github.com/hislordshipprof/Hello-Microverse/blob/0091dd5bb94d31a0b9c12f3f57a5b7bc7b9b54aa/README.md?plain=1#L61

Please change this to a header and provide some actual acknowledgments. You can mention Microverse and/or the Figma template.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @hislordshipprof,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**